### ISSUE TITLE
multi-wiggle track

### ISSUE THREAD

**Issue by rbuels** [GMOD/jbrowse-components#842]
Make a new track type that contains multiple wiggle tracks, displays them all as subtracks, and has powerful features for working with multiple wiggle datasets.

Most fundamental is displaying all the datasets with the same Y scale (or heat colors or whatever) so that they can be easily compared visually.

**Comment by cmdcolin** [comment]
I wanted to see what could maybe be done for this as I think there is interest in it (xref https://github.com/GMOD/jbrowse-components/discussions/2943 https://github.com/GMOD/jbrowse-components/discussions/2865 etc)

Still trying to figure out the right type of config and code organization for it, little tricky

Currently, thinking that we will have a but of a parallel universe to single wiggle tracks with MultiQuantitativeTrack, MultiLinearWiggleDisplay, and MultiXYRenderer. Will also have something called a MultiWiggleAdapter that can take multiple wiggle subadapters and collectively get stats on them

This parallel universe does have a bit of copy copying and pasting, but it does have somewhat unique concerns so I think it might be ok to be this way.

This takes care of the "config"  but how does someone actually interactively create a multiquantiativetrack in the GUI? I envision we could put the track selector into an alternative "multi-select mode" to allow multiple single quantative to be combined into a multibigwig track as a session track. The single track's adapter (and possibly rendering options, but probably just adapters for now) would get stored into the multiquantativetrack's subadapter array

Example

```
{
      "type": "MultiQuantitativeTrack",
      "trackId": "volvox_microarray_multi",
      "name": "MultiWig",
      "assemblyNames": ["volvox"],
      "adapter": {
        "type": "MultiWiggleAdapter",
        "subadapters": [
          {
            "source": "k1",
            "type": "BigWigAdapter",
            "bigWigLocation": {
              "uri": "v1.cram.bw"
            }
          },
          {
            "source": "k2",
            "type": "BigWigAdapter",
            "bigWigLocation": {
              "uri": "v2.cram.bw"
            }
          },
          {
            "source": "k3",
            "type": "BigWigAdapter",
            "bigWigLocation": {
              "uri": "v3.cram.bw"
            }
          },
          {
            "source": "k4",
            "type": "BigWigAdapter",
            "bigWigLocation": {
              "uri": "v4.cram.bw"
            }
          }
        ]
      }
    },

```

![Screenshot from 2022-06-15 10-29-10](https://user-images.githubusercontent.com/6511937/173879286-51b23dea-9016-4bc5-8fb4-55c477547a5d.png)

colors are hardcoded here. note also that it is tempting to put e.g. the colors in the MultiWiggleAdapter config, next to where the tracks are displayed, which may work (the renderer receives the adapterConfig for example), but it might not really be the best option for the app (reasons: adapter is fully re-created from scratch every time you update the adapter config at all, and most tracks don't config display/renderer features using adapter). in above screenshot colors are auto-assigned from a palette

**Comment by cmdcolin** [comment]
fixed by #3043 :partying_face:
